src/helpers.ts

```diff
@@ -1667,8 +1702,14 @@ Note that in this case, the contract deployment will not behave the same if depl
       from = wallet.address;
     } else {
       if (availableAccounts[from.toLowerCase()]) {
-        ethersSigner = provider.getSigner(from);
-      } else {
+        if(network.zksync){
+          ethersSigner = provider.getSigner(from);
+        } else {
+          network.provider.getSignerFrom(from)
```

Comment:
Is this line a mistake? It doesn't look like it does anything and I'm getting the error:

```
TypeError: network.provider.getSignerFrom is not a function
```

Reply:
Hi @janjakubnanista ,

It seems that a previous commit I didn’t commit contained a part I missed, which turned out to be incorrect. Here's the relevant link to that commit:
[GitHub Commit](https://github.com/wighawag/hardhat-deploy/commit/cfad6a7ba753c061a2561df9de954addbdafe8ad#diff-ca5696b367d474e785317cb7a0d9853fef5729387ab8d0fab4c46268c03aae99R1713)

Reply:
I've created an issue to track this: #558

Reply:
There are more issues actually so I reverted this PR as I don't have time right now to look deeper
but after fixing the line you mention @guidanoli I found this other issue : https://github.com/wighawag/hardhat-deploy/commit/7ac8fa1d983323f065c789c7523334fea47fa19a#r148088322

if you can fix and make sure there are no other issue I can bring back the PR, Thanks!